.gitignore

```diff
@@ -1,2 +1,6 @@
 VOC/
 __pycache__/
+.idea/
+models/checkpoints/
+training/VOC/
+/od_from_scratch/model_checkpoints/
```

Comment:
/od_from_scratch is not part of the repository.
you can just remove it.

Reply:
done